content/en/blog/2021-08-15-seccomp-default.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-15
```

Comment:
@PI-Victor @pmmalinov01 @ashnehete @carlisia @chrisnegus @ritpanjw how do we decide about the date?

Reply:
Heya, i'm not sure at this point, but we do have a sig-docs blog meeting today and i can bring this up then, do you have any preferred date?

Reply:
There's usually been a separate group looking at the post-release comms including blog articles, tweets, more things. I assume there still would be, and if there is then they can set dates.

I'd expect to have no more than 3 new feature articles per week, with at least a 1 day gap between them (eg Mon, Weds, Fri).

Reply:
Hi all. Yes as @sftim mentioned, there's a planned cadence of 3 feature blogs a week post-release. One or two may be scheduled on the same day as the release.

The publish dates are on the [1.22 Feature Blog tracking sheet](https://docs.google.com/spreadsheets/d/1hgFWig6YBSYORP3huwEyU0VlXGcYi6wMxjYW-swzCIY), and this one is currently scheduled for publication with the release on August 4.

Let us know if this needs a change, otherwise you can set it for then. Thanks!

Reply:
Actually, looking back it doesn't appear that we double up like this. @sftim  I'd like your input, should we update the schedule so that we have release blog on its own, then a feature blog on the three-business-day cadence from there?

Reply:
Changed the date to `2021-08-04` for now.

Reply:
> Changed the date to `2021-08-04` for now.

2021-08-04 is the scheduled release day for v1.22 and the release blog. I defer to @sftim and the 1.22 Comms team on the release date: @jlbutler , @skrishna-unix , @kunal-kushwaha , @rajula96reddy

Reply:
Hi @saschagrunert , one more quick change thanks. Based on some feedback we've reworked the blog schedule.

Please set 2021-08-20 as the publication date, based on the updated feature blog tracking sheet data. Thank you!

Reply:
Changed it to `2021-08-20`. :+1: